### PR TITLE
20 - Doors and level game

### PR DESCRIPTION
**Pull request by gNage**

### CODE REVIEW

File: HW1/Evgeniy_Nikulin/Doors_and_levels_game/Doors_and_levels_game/Program.cs [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+    class Program
+    {
+        static Game game = new Game();
+        static void Main(string[] args)
+        {
+            game.Start();
+        }
+    }
+
+    public class Game
```
Comment (spore9) [inline HW1/Evgeniy_Nikulin/Doors_and_levels_game/Doors_and_levels_game/Program.cs]: In my opinion, relocating Game class in a separate file will improve readability.

File: HW1/Evgeniy_Nikulin/Doors_and_levels_game/Doors_and_levels_game/Program.cs [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+        ulong GetDoor()
+        {
+            ulong num;
+            string str;
+
+            while (true)
+            {
+                Console.Write("Your choose: ");
+                str = Console.ReadLine();
+                try
```
Comment (spore9) [inline HW1/Evgeniy_Nikulin/Doors_and_levels_game/Doors_and_levels_game/Program.cs]: In my opinion, try-catch construction will work slower, than using of UInt64 function try parse and checking of its result.

Comment (gNage) [reply HW1/Evgeniy_Nikulin/Doors_and_levels_game/Doors_and_levels_game/Program.cs]: sure, slower, for 5 milliseconds when we parse not a number
Безымянный

File: HW1/Evgeniy_Nikulin/Doors_and_levels_game/Doors_and_levels_game/Program.cs (hunk 1 above)
Comment (gNage) [reply HW1/Evgeniy_Nikulin/Doors_and_levels_game/Doors_and_levels_game/Program.cs]: i know but this is not necessarily for this issue. Many files may just confuse

**Comment by vladimirchr** [comment]
Cool, i like your solution. The only thing i would change is 2 variable naming 
`ulong num;
 string str;` Even though they are not used for long, it's more clear when they have more meaningful name.

**Pull request merged**